test/e2e/storage/drivers/csi.go

```diff
@@ -348,6 +354,9 @@ func (g *gcePDExternalCSIDriver) GetDriverInfo() *DriverInfo {
 func (g *gcePDExternalCSIDriver) SkipUnsupportedTest(pattern testpatterns.TestPattern) {
 	framework.SkipUnlessProviderIs("gce", "gke")
 	framework.SkipIfMultizone(g.driverInfo.Framework.ClientSet)
+	if pattern.FsType == "xfs" {
```

Comment:
is there a way to put this check in the "testsuite" skip checks instead of the driver skip tests? This skip "holds" for *any* driver that runs this test suite.

Reply:
@davidz627 

> is there a way to put this check in the "testsuite" skip checks instead of the driver skip tests? This skip "holds" for any driver that runs this test suite.

I think that moving this logic to [```skipUnsupportedTest```](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/testsuites/base.go#L87) will make xfs tests skip for any drivers and any testsuites. However, to do so, we should blacklist unsupported distros instead of whitelisting just ```"ubuntu"``` and ```"custom"```, because we are not sure which distro will be passed here. (In current code, it assumes that this code path in gce driver's ```SkipUnsupportedTest``` reaches by only gce tests, so just whitelisting them is working fine.)

So, could you let me know which distro to blacklist?
(I guess that ```"gci"``` is needed here and how about ```"debian"``` and ```"coreos"```?)

Reply:
@davidz627 

commit 77784a6 will do above, assuming that only ```"gci"``` should be blacklisted. If this isn't true, please point it out.
(Currently, xfs test seems running only on gce CI, so we can go ahead once we confirm that it is true in gce CI, and add blacklist when we need to add some more latter.)

Reply:
@msau42 @lookuptable do either of you know the answer to this? I am only aware of `gci` not having `xfs` support but there could be more

Reply:
For gce CI, blacklisting "gci" is fine, although we need to find a way to deal with "custom", which can be both ubuntu or gci.  Can we check for the presence of xfs tools on the node?